libraries/src/Form/Field/MediaField.php

```diff
@@ -310,8 +310,8 @@ public function getLayoutData()
             /**
              * This is the case where a folder is configured in directory attribute of the form field. The directory needs
              * to be a relative folder of the folder configured in Path to Images Folder config option of Media component.
-             * Same with a already stored local image above, we need to validate and make sure top level folder is one of the directory
-             * configured in filesystem local plugin
+             * Same with an already stored local image above, we need to validate and make sure top level folder is one of the 
```

Comment:
```suggestion
             * Same with an already stored local image above, we need to validate and make sure top level folder is one of the
```
Fix PHPCS failing in Drone.